src/main/java/com/stage/users/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package com.stage.users.exception;
+
+
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus
```

Comment:
There is not much to refactor in this code. One minor suggestion would be to add the @ResponseStatus annotation with an appropriate HTTP status code : 
@ResponseStatus(HttpStatus.NOT_FOUND)